install/install.py

```diff
@@ -11,174 +11,198 @@
-
-    def install_using_pip(self, requirements_file):
-        # Instead of using file directly with pip which can crash because of single library
-        self.run_command("sudo -E pip2 install --upgrade -r %s" % requirements_file)
-
-    def install_restricted_from_cfg(self, config_file):
-        cp = ConfigParser.ConfigParser({"RootDir":self.RootDir, "Pid":self.pid})
-        cp.read(config_file)
-        for section in cp.sections():
-            Colorizer.info("[*] Installing %s" % section)
-            self.install_in_directory(os.path.expanduser(cp.get(section, "directory")), cp.get(section, "command"))
-
-    def install(self, cmd_arguments):
-
-        args = self.parser.parse_args(cmd_arguments)
-
-        # User asked to select distro (in case it cannot be automatically detected) and distro related stuff is installed
-        cp = ConfigParser.ConfigParser({"RootDir":self.RootDir, "Pid":self.pid})
-        cp.read(self.distros_cfg)
-
-        #Try get the distro automatically
-        distro, version, arch = platform.linux_distribution()
-        distro_num = 0
-        if "kali" in distro.lower():
-            distro_num = 1
-        elif "samurai" in distro.lower():
-            distro_num = 2
-        elif "debian" in distro.lower():
-            distro_num = 3
-
-        # Loop until proper input is received
-        while True:
-            if distro_num != 0:
-                Colorizer.info("[*] %s has been automatically detected... " % distro)
-                Colorizer.normal("Continuing in auto-mode")
-                break
+        return False
 
-            if args.no_user_input:
-                distro_num = 0
-                break
-
-            print("")
-            for i, item in enumerate(cp.sections()):
-                Colorizer.warning("(%d) %s" % (i + 1, item))
-            Colorizer.warning("(0) My distro is not listed :( %s" % distro)
-
-            distro_num = raw_input("Select a number based on your distribution : ")
-            try: # Cheking if valid input is received
-                distro_num = int(distro_num)
-                break
-            except ValueError:
-                print('')
-                Colorizer.warning("Invalid Number specified")
-                continue
-
-        # First all distro independent stuff is installed
-        self.install_restricted_from_cfg(self.restricted_cfg)
 
+def run_command(command):
+    """Execute the provided shell command.
+
+    :param command: (~str) Linux shell command.
+
+    :return: True - if command executed, and False if not.
+    """
+    Colorizer.normal("[*] Running following command")
+    Colorizer.info("%s" % command)
+    # If command is `wget`, then before execution, `wget_wrapper` checks whether there is enough disk space available
+    if not wget_wrapper(command):
+        return False
+    return os.system(command)
+
+
+def zest_version():
+    """Prints the local git repo's last commit hash."""
```

Comment:
Changed to `get_zest_last_commit_hash()` - can't think anything better.. :smile: